Sources/CoffeeBot/CoffeeManager/CoffeeManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoffeeManager.swift
+//  SwiftBot
+//
+
+import Foundation
+import BotsKit
+import LoggerAPI
+
+internal protocol CoffeeManagerProtocol {
+    func process(activity:Activity)
+}
+
+class CoffeeManager {
```

Comment:
I thought of CofeeBot as a main.m which speaks with the external modules and this CoffeeManager handles all the internal services and everything.